src/main.ts

```diff
@@ -22,35 +23,39 @@ function getName(path: string): string {
     return path.substring(path.lastIndexOf('/') + 1, path.indexOf('.vue'));
 }
 
-export default {
-    install: (app: App, setting: DeepPartial<Settings> = {}): void => {
-        app.config.globalProperties.Vueish = {
-            clearableByDefault: setting.hasOwnProperty('clearableByDefault')
-                ? setting.clearableByDefault
-                : defaultSettings.clearableByDefault,
-            icons: Object.assign({}, defaultSettings.icons, setting.icons ?? {})
-        } as Settings;
-
-        if (globalThis.document && getComputedStyle(document.body).position !== 'relative') {
-            document.body.style.position = 'relative';
+export function createUILibrary(setting: DeepPartial<Settings> = {}): Plugin {
```

Comment:
a) What prompted this change? 
b) How does this change the usage in the apps?

Reply:
It changes it by instead of importing in `import Vueish from '@karnama/vueish` it is now `import { createLibrary } from '@karnama/vueish`;
Before the import triggered an error
```
TS1192: Module '"/Users/nandorkraszlan/Web/woosh-web/node_modules/@karnama/vueish/types/index"' has no default export.
```
desite having a module declaration
The import now doesn't trigger the error.

Reply:
Fair enough. Not having that error myself but if the library continues to work then 🤷